modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AbstractIndexCommandParams.java

```diff
@@ -41,6 +47,20 @@ public String schemaName() {
         return schema;
     }
 
+    /**
+     * Returns table name.
+     */
+    public String tableName() {
+        return tableName;
+    }
+
+    /**
+     * Returns {@code true} if index is unique, {@code false} otherwise.
+     */
+    public boolean isUnique() {
```

Comment:
we either need to add prefix to every getter or don't use it at all within *Params classes

Reply:
agree, I'll fix it without a prefix.

Reply:
fix it